src/_guides/language/language-tour.md

```diff
@@ -4202,9 +4202,9 @@ Here are some cases when you might use deferred loading:
 * To load rarely used functionality, such as optional screens and dialogs.
 
 {{site.alert.warn}}
-  **Only dart2js supports deferred loading.**
-  Flutter, the Dart VM, and dartdevc don't support deferred loading.
-  For more information, see
+  **Only `dart compile js` supports deferred loading.**
```

Comment:
FYI - I believe some support for deferred loading has been added to the VM and we are working towards adding it to the dartdevc workflow (webdev). That's tangential to this change, so I wouldn't change anything here, just wanted to give you a heads up :)